### ISSUE TITLE
Dashboards not working in Grafana.

### ISSUE THREAD

**Issue by larkinis** [openiomon/fos2graphite#7]
Hello!
Many charts don't work and don't show data in dashboards in Grafana. Uploaded dashboards according to the instructions from the /opt/fos2graphite/dashboards/graphite/ folder in Grafana.
Grafana version: v10.0.1, but tried on many versions 8.1.6 and 8.2.7. Still doesn't work

**Comment by mamoep** [comment]
Which data source are you using? Some of the panels are built with VictoriaMetrics MetricsQL specific functions which are not available in native Prometheus.

**Comment by larkinis** [comment]
Data is collected by graphite
![Screenshot_3](https://github.com/openiomon/fos2graphite/assets/84312745/daef181a-f6d5-4bfd-83d3-3215ddcdbfe1)

**Comment by mamoep** [comment]
Can you check if all the panels use the correct data source?

**Comment by larkinis** [comment]
Yes, everywhere in Dashboards datasource by default is Graphite. I checked.

**Comment by mamoep** [comment]
Do you get any errors in the query inspector?

**Comment by larkinis** [comment]
mamoep,
No, i think, there are no errors in query inspector. 
![Screenshot_4](https://github.com/openiomon/fos2graphite/assets/84312745/f94fb2da-dcc1-4e7a-8a60-62a8707fbd5d)

These Dashboards for monitoring brocade work for you? Or you didn't tested them? :)

**Comment by larkinis** [comment]
You can offer any working version of Grafana. Working Dashboards are important to us.
Installing any version using docker and importing Dashboards is not difficult. :)

Also, most of the charts is deprecated. Such as:
Graph(old), and Panel plugin not found: grafana-piechart-panel use in Brocade: Initiator / Target Details (Realtime) and Brocade: Port Status (Realtime)

**Comment by mamoep** [comment]
The function nonNegativeDerivative() is used to convert an ever increasing value to a rate. See the documentation: https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.nonNegativeDerivative

Can you check if your graphite api implementation does support this function? "perSecond" could also be used instead.

**Comment by mamoep** [comment]
I updated the Graphite Dashboards to get rid of the deprecated panels. You can fetch them from the repo, new release will follow later.
